src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/NServiceBus.Transport.RabbitMQ.AcceptanceTests.csproj

```diff
@@ -11,9 +11,11 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="[15.*,16)" />
+    <PackageReference Include="NServiceBus" Version="[7.0.0-*,8)" PrivateAssets="None" />
```

Comment:
While NServiceBus is pre-release, this matches the dependency declared in the package project, but when NServiceBus is stable, the package will declare `[7.0.0, 8)` and the test will declare `[7.*, 8)`, as per [the guidelines](https://github.com/Particular/NServiceBus/blob/develop/guidelines/coding-guidelines.md#core).